rdt-open-reader/src/main/java/io/ona/rdt/activity/PatientRegisterActivity.java

```diff
@@ -209,6 +212,14 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
                 return true;
             }
         });
+
+        long lastSyncTimestamp = RDTApplication.getInstance().getContext().allSharedPreferences().getPreferences().getLong(CovidConstants.SyncPreference.LAST_SYNC_DATE, 0);
+        if (lastSyncTimestamp > 0) {
+            Date lastSyncDate = new Date(lastSyncTimestamp);
+            String lblSync = getString(R.string.drawer_menu_item_sync) + ": %1$s";
+            MenuItem syncMenuItem = menuNav.findItem(R.id.menu_item_sync);
+            syncMenuItem.setTitle(String.format(lblSync, new SimpleDateFormat("hh:mm a, MMM dd", org.smartregister.util.Utils.getDefaultLocale()).format(lastSyncDate)));
```

Comment:
Why not use `Locale.getDefault()`. `org.smartregister.util.Utils.getDefaultLocale()` is too long.

Reply:
done